paddle/fluid/pybind/pybind.cc

```diff
@@ -1245,6 +1245,12 @@ All parameter, weight, gradient are variables in Paddle.
           [](BuildStrategy &self, BuildStrategy::ReduceStrategy strategy) {
             PADDLE_ENFORCE(!self.IsFinalized(), "BuildStrategy is finlaized.");
             self.reduce_ = strategy;
+#ifdef WIN32
+            if (self.reduce_ == BuildStrategy::ReduceStrategy::kReduce) {
+              LOG_FIRST_N(WARNING, 1) << "Windows support AllReduce only.";
```

Comment:
Do you know the difference between of Reduce and AllReduce mode?

Reply:
Got it